src/functions/lists.rs

```diff
@@ -61,10 +61,8 @@ pub fn register(f: &mut BTreeMap<&'static str, SassFunction>) {
                     sep1.or(sep2).unwrap_or(ListSeparator::Space)
                 }
                 ref other => {
-                    return Err(Error::badarg(
-                        "'comma', 'space', or 'auto'",
-                        other,
-                    ))
+                    Err(Error::badarg("'comma', 'space', or 'auto'", other))
+                        .unwrap()
```

Comment:
Why is this `return Err(...)` changed to `Err(...).unwrap()`?  This changes the behaviour from returning the error to panicking with it, which is probably not a good thing?

Reply:
On a previous commit, the ci stucked on that at `cargo fmt -- --check`, if I remember correctly.
Now it seams to work. Just reverted it. :)